corehq/apps/domain/models.py

```diff
@@ -67,24 +64,6 @@
         lang_lookup[lang['two']] = lang['names'][0]
 
 
-class DomainMigrations(DocumentSchema):
-    has_migrated_permissions = BooleanProperty(default=False)
-
-    def apply(self, domain):
-        if not self.has_migrated_permissions:
-            logging.info("Applying permissions migration to domain %s" % domain.name)
-            from corehq.apps.users.models import UserRole, WebUser
-            UserRole.init_domain_with_presets(domain.name)
```

Comment:
Does this line still get called when a new domain is created? I think that was the only part of this that still has a function.